interop/tapir/src/main/scala/caliban/interop/tapir/TapirAdapter.scala

```diff
@@ -230,18 +231,14 @@ object TapirAdapter {
     makeHttpUploadEndpoint.serverLogic(logic)
   }
 
-  def makeWebSocketEndpoint[R, E](implicit
+  def makeWebSocketEndpoint(implicit
     inputCodec: JsonCodec[GraphQLWSInput],
     outputCodec: JsonCodec[GraphQLWSOutput]
-  ): PublicEndpoint[ServerRequest, TapirResponse, CalibanPipe, ZioStreams with WebSockets] = {
-    val protocolHeader = Header("Sec-WebSocket-Protocol", "graphql-ws")
+  ): PublicEndpoint[ServerRequest, TapirResponse, CalibanPipe, ZioStreams with WebSockets] =
```

Comment:
Do we make it configurable or do we support both? It looks like you supported both in the Tapir Adapter but made it configurable in zio-http?

Reply:
Yeah there was a subtle difference in how the two were used. Since the zio-http doesn't get the request context, so I pushed it up to the user. I guess I could force them to pass the request?

Reply:
There's also an issue that I think I might still need to address, where certain errors need to immediately kill the connection with an error.

Reply:
If you can I guess it's best if there's nothing to configure.